src/main/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtils.java

```diff
@@ -54,22 +54,25 @@ public static BigQuery getBigQueryEndPoint(String executionProjectId) {
      * Will block until results are returned.
      * For more information on querying BigQuery tables, see: https://cloud.google.com/bigquery/sql-reference/
      * @param queryString The {@link BigQuery} query string to execute.  Must use standard SQL syntax.  Must contain the project ID, data set, and table name in the `FROM` clause for the table from which to retrieve data.
+     * @param labels The {@link BigQuery} label to add the job run.  Must use Map<String, String>.  Must contain Job label and gatk_execution_id.
```

Comment:
Can you remove the "Must contain Job label and gatk_execution_id"?  While that's true for our usages, it's not true for this lower level function (ie this can be called from anywhere, not just variant store work).  But also maybe add the statement "Can be null to indicate no labels" if that's true (I think it is right?)